src/dialects/abstract/query-interface.types.ts

```diff
@@ -0,0 +1,10 @@
+import type { QueryRawOptions, Sequelize } from 'src/sequelize';
+import type { AbstractQueryGenerator, CreateSchemaQueryOptions } from './query-generator';
+
+export interface QueryInterfaceOptions {
+  sequelize: Sequelize;
+  queryGenerator: AbstractQueryGenerator;
+}
+
+export interface CreateSchemaOptions extends CreateSchemaQueryOptions, QueryRawOptions {}
```

Comment:
I think these types could be documented with something like this for our API reference website:

```
/** Options accepted by {@link AbstractQueryInterfaceTypeScript#createSchema} */
```